### PR TITLE
added camel-k operator

### PR DESCRIPTION
**Pull request by durandom**
to the smaug cluster

I clicked install at https://operatorhub.io/operator/camel-k and added the contents of https://operatorhub.io/install/camel-k.yaml

anything more needed?

fixes https://github.com/operate-first/support/issues/476

### CODE REVIEW

**Review by 4n4nd — CHANGES_REQUESTED** [review]
1. The subscription needs to be added in the [base](https://github.com/operate-first/apps/tree/master/cluster-scope/base/operators.coreos.com/subscriptions).
2. The patch generally contains the stuff that is different from the base, in case of operator subscriptions it is usually the `spec.channel` that is different.

File: cluster-scope/base/operators.coreos.com/subscriptions/camel-k/subscription.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: camel-k
+  namespace: operators
```
Comment (tumido) [inline cluster-scope/base/operators.coreos.com/subscriptions/camel-k/subscription.yaml:5]: This namespace doesn't exist. I think you wanted to use `openshift-operators` instead. It's also generally better to enforce the namespace via `kustomization.yaml` (see this [Kiali example here](https://github.com/operate-first/apps/blob/master/cluster-scope/base/operators.coreos.com/subscriptions/kiali-operator/kustomization.yaml)).

File: cluster-scope/base/operators.coreos.com/subscriptions/camel-k/subscription.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+spec:
+  channel: stable-1.6
+  installPlanApproval: Automatic
+  name: camel-k
+  source: operatorhubio-catalog
```
Comment (tumido) [inline cluster-scope/base/operators.coreos.com/subscriptions/camel-k/subscription.yaml:10]: I don't think we have such catalog available here. In operator hub we have 2 options how to install camel-k:

1. comminuty version of the operator named `camel-k` from `community-operators` catalog
2. redhat certified version with name `red-hat-camel-k` from `redhat-operators` catalog

Based on channel selected I think you wanted to use `community-operators` catalog.

You may also consider updating the channel to `stable-1.7`


These are your options (package name and the catalog can be seen in the URL)

Either community operator:
image

Or the Red Hat version of it:
image

**Review by tumido — CHANGES_REQUESTED** [review]

**Comment by durandom** [comment]
@tumido thanks 💯 that was a decent review. I addressed the comments

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Pull request merged**